samples/openthread/cli/src/low_power.c

```diff
@@ -8,7 +8,9 @@
 #include <zephyr/net/openthread.h>
 #include <zephyr/device.h>
 #include <zephyr/pm/device.h>
+#ifdef CONFIG_RAM_POWER_DOWN_LIBRARY 
```

Comment:
```suggestion
#if defined(CONFIG_RAM_POWER_DOWN_LIBRARY)
```
To keep consistency with `main.c`